### PR TITLE
Add example for CSP nonce & Jinja template integration

### PR DESCRIPTION
**Pull request by moser**
First some more docs & integration for the CSP nonce feature.
If that's OK, I will create a release PR later.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -248,6 +248,33 @@ example CSP, this site uses the setting specified by the ``default-src``
...
+A web site administrator wants to allow embedded scripts (which might
+be generated dynamicially).
+
+.. code:: python
+
+    csp = {
+        'default-src': '\'self\'',
+        'script-src': '\'self\'',
+    }
+    talisman = Talisman(app,
```
Comment (jonparrott) [inline README.rst]: nit: can you line break at the opening `(` to avoid hanging indents?

Comment (moser) [reply README.rst]: You mean
```python
foo = bar(
  arg=1
)
```

instead?

Comment (jonparrott) [reply README.rst]: yea

**Comment by jonparrott** [comment]
Thanks, @moser!

**Pull request merged**